tests/scripts/all.sh

```diff
@@ -2190,6 +2217,31 @@ component_release_test_valgrind_constant_flow_psa () {
     make memcheck
 }
 
+component_release_test_valgrind_constant_flow_psa_no_asm () {
```

Comment:
The places where PSA vs not-PSA makes a difference, and the places where asm vs not-asm makes a difference, are separate. So we don't really gain anything by testing all four combinations. Since this component is very slow, we prefer to minimize it. So please remove the `psa_no_asm` combination.